src/app/solicitation/summary/results-detail/results-detail.component.html

```diff
@@ -20,8 +20,19 @@
             <div style="padding-bottom: 5px"><a target="_blank" href="{{solicitation.url}}">{{solicitation.url}}</a></div>
             <div *ngIf="solicitation.reviewRec !== 'Not Applicable' " style="padding-bottom: 5px">
               <span style="font-weight:600">Results:&nbsp;</span>
-              <span style="font-weight:500" >{{solicitation.reviewRec}}</span>
-
+              <span style="font-weight:500" >{{solicitation.reviewRec}}</span>                    
+            <!-- insert button here for linking to ART -->
+            </div>
+            <div class="row">
+              <div class="col-md-8">
+                <a href="https://section508.gov/art" target="_blank">
+                  <div id="art" class="srt-button" style="width: 150px; " >
```

Comment:
You can add the width to the art id CSS entry. Additionally, if you wanted the color to be different, we could utilize the bootstrap buttons, and changed the color scheme with the ID stylization.

Reply:
added the width to the css and removed from html